.github/workflows/scripts/install_python_client.sh

```diff
@@ -16,7 +16,7 @@ cd "$(dirname "$(realpath -e "$0")")"/../../..
 
 pip install twine wheel
 
-export REPORTED_VERSION=$(http $PULP_URL/pulp/api/v3/status/ | jq --arg plugin galaxy --arg legacy_plugin galaxy_ng -r '.versions[] | select(.component == $plugin or .component == $legacy_plugin) | .version')
+export REPORTED_VERSION=$(http {$PULP_URL}{$PULP_API_ROOT}api/v3/status/ | jq --arg plugin galaxy --arg legacy_plugin galaxy_ng -r '.versions[] | select(.component == $plugin or .component == $legacy_plugin) | .version')
```

Comment:
I am trying to resolve CI issues with 404s on `/pulp/api/v3/status/`. @fao89 do we need to update the calls to the status api in these search results to include `settings.API_ROOT`? https://github.com/pulp/plugin_template/search?q=%22api%2Fv3%2Fstatus%22&type=

Reply:
the plugin-template already brings the necessary changes
https://github.com/pulp/plugin_template/blob/f49381dbd6aa45d2a510235a285a7bc3e5274eff/templates/github/.ci/ansible/start_container.yaml#L78-L90

Reply:
FYI, S3 job already runs rerouted:
https://github.com/ansible/galaxy_ng/blob/master/.github/workflows/scripts/install.sh#L126-L127
https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/tests/functional/utils.py#L245-L246

Reply:
@fao89 I am using https://github.com/pulp/plugin_template/commit/36696fcf9dfe3c5b418ed498c4f97d3168361e5e but in the `Galaxy CI / test (pulp)` job logs we still see
```
2022-05-11T13:07:02.3709391Z http: error: SSLError: HTTPSConnectionPool(host='pulp', port=443): Max retries exceeded with url: /pulp/api/v3/status/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1131)'))) while doing a GET request to URL: https://pulp/pulp/api/v3/status/
```

Reply:
actually plugin-template needs more changes for your use case, I'll start a PR

Reply:
you need this: https://github.com/pulp/plugin_template/pull/641